### DISCUSSION TITLE
How to style the when I use the "Access the data as tiles"

### DISCUSSION THREAD

**Discussion by Sverre-Iversen** [ldproxy/ldproxy#777] [Configuration] [answered]
Hi,

Customized styling works when choosing (see image below) "Access the data" and "Styles to render the data in maps".
When I try **"Access the data as tiles**", the default styling (blue dots an polygons) are shown. 
**Any way to use customized styling here?** 
 
![link-to main-resources](https://user-images.githubusercontent.com/1905113/201652158-c9db269b-d4ac-433f-a21a-d3fa21c5c0f8.png)

Kind Regards 
Sverre Iversen

**Comment by cportele** [comment, marked as answer]
Hi Sverre,

yes (see https://demo.ldproxy.net/daraa/tiles for an example). 

In both TILES and FEATURES_HTML you can configure a style using the "style" option. I assume, you have configured "style" in FEATURES_HTML, but not in TILES?

In addition you can set a "defaultStyle" in the HTML building block.  This style is then used for both FEATURES_HTML and TILES as the style, if no style is configured explicitly. So, in general, if the same style should be used everywhere, it should be sufficient to configure the "defaultStyle" in HTML.

In FEATURES_HTML and TILES you can also override the default style to use the blue wireframe style with "style: NONE".

Best regards, Clemens

**Reply by Sverre-Iversen** [reply]
Thanks, I'll try it out
